stroeer/core/v1/article.proto

```diff
@@ -895,29 +897,59 @@ message Article {
     float score = 3;
   }
   /** @CodeBlockEnd */
+  /**
+   *
+   * ## `RelatedArticle`
+   *
+   * An editorial article, which is related to another article.
+   * Related articles are specified manually be the editorial department or
+   * identified by recommendation systems.
+   *
+   * | Field name             | Type   | Description                                                    |
+   * |------------------------|--------|----------------------------------------------------------------|
+   * | article                | stroeer.core.v1.Article | Data of the related article. May be empty, if the article was not resolved to an article yet. This message will only contain data required to render the article as a _Teaser_ (e.g. with `Article.body` set to `null`. thus not containing any data that is only required on detail pages).|
+   * | related_article_source | string | Source of the related article.                                 |
+   * | id                     | int64  | id of the unresolved related article                           |
+   *
+   * @CodeBlockStart protobuf
+   */
+
+  message RelatedArticle {
+    // Data of the related article. This message will only contain data required
+    // to render the article as a _Teaser_ (e.g. with `Article.body` set to `null`
+    // thus not containing any data that is only required on detail pages).
+    // This field may also be empty if the id was not resolved to an article yet.
+    stroeer.core.v1.Article article = 1;
+    // Source of the related article.
+    string related_article_source = 3;
+
+    // id of the unresolved related article
+    int64 id = 4;
```

Comment:
maybe just save `Article.id` in the `article` above at transformation time and replace with teaser in the API instead of adding this extra field?

Reply:
@thisismana  was hälst du davon?